data-ingestion/kfp/templates/execute-website-ingestion-kubeflow-pipeline.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
```

Comment:
Minor request, do you mind making a folder called `task` and putting this and the other task in there?

Reply:
This request is implemented.